keyboards/ergodox_ez/keymaps/danielo515/rules.mk

```
@@ -0,0 +1,9 @@
+# Set any rules.mk overrides for your specific keymap here.
+# See rules at https://docs.qmk.fm/#/config_options?id=the-rulesmk-file
+EXTRAFLAGS += -flto
```

This can be replaced by: 
```suggestion
LTO_ENABLE = yes
```